src/mmw/sass/pages/_analyze.scss

```diff
@@ -216,7 +216,7 @@
       border-radius: $radius-sm;
       padding: 1rem;
 
-      > .analyze-model-package-link {
+      > a {
```

Comment:
This was changed because each `analyze-model-package-link` is handled by the JavaScript to create a project with the given model package, but we want a button that looks like that but doesn't work like that. Hence the switch to a generic `a` tag.